packages/form-field/src/FormField/FormField.styles.ts

```diff
@@ -1,6 +1,4 @@
 import { css } from '@leafygreen-ui/emotion';
-import { Theme } from '@leafygreen-ui/lib';
-import { palette } from '@leafygreen-ui/palette';
 import { BaseFontSize, Size, spacing, typeScales } from '@leafygreen-ui/tokens';
 
 export const getFontSize = ({
```

Comment:
I know this isn't changed in this PR, but this Fn name should be `getFontSizeStyles` (as-is it seems like the func would return just a number)